### PR TITLE
Civ 3768 some useful info about help fees

### PR DESCRIPTION
**Pull request by GorilaNaranja**
### JIRA link ###
https://tools.hmcts.net/jira/browse/CIV-3768

### Change description ###
Some useful information about Help with Fees

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by hmcts-version1-domini — APPROVED** [review]

File: src/main/modules/i18n/locales/cy.json [hunk 1]
```diff
@@ -596,6 +596,13 @@
...
+      "HELP_WITH_FEES": "If you want to apply for Help with Fees, you will complete an online form in a new window. This will give you a reference number. Please note the number and keep it safe, as you will need it later in the claim process. (NB: if you have more than one claim going on, each claim must have a separate Help with Fees application and reference number). You can send in your claim as soon as you have the Help with Fees reference number, but the claim will not be issued until the Help with Fees application has been processed by the court. This is so that you will know what fees are covered and what you still have to pay. If you choose not to pay, the claim will not be issued. The claim will stop here unless you change your mind within 95 days.",
+      "HELP_WITH_FEES_ADVICE": "You may be asked by the court to email evidence in support of your application before your claim is issued.",
+      "CONTINUE_HELP_WITH_FEES": "Do you wish to continue to make a Help with Fees Application?"
```
Comment (portizdev) [inline src/main/modules/i18n/locales/cy.json:604]: Sorry about that but.. alphabetic order haha 😉

File: src/main/views/features/public/eligibility/some-useful-info-fees.njk [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+{% from "../../../macro/yesNoForm.njk" import yesNoRadioButton %}
+{% from "../../../macro/page-title.njk" import setPageTitle %}
+{% from "../../../macro/error-summary.njk" import errorSummary %}
+
+
+ELIGIBILITY_USEFUL_INFO_FEES": {
+      "TITLE": "Some useful information about Help with Fees",
+      "HELP_WITH_FEES": "Help with Fees...",
+      "CONTINUE_HELP_WITH_FEES": "Do you wish to continue to make a Help with Fees Application"
+    },
```
Comment (portizdev) [inline src/main/views/features/public/eligibility/some-useful-info-fees.njk:14]: This must be here?

File: src/test/unit/views/features/public/eligibility/some-useful-info-fees.test.ts [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+import request from 'supertest';
+import {app} from '../../../../../../main/app';
+import {ELIGIBILITY_INFORMATION_FEES_URL} from '../../../../../../main/routes/urls';
+import {TestMessages} from '../../../../../utils/errorMessageTestConstants';
+
+const jsdom = require('jsdom');
+const {JSDOM} = jsdom;
+jest.mock('../../../../../../main/modules/oidc');
+
+describe('Eligibility Tenancy Deposit View', () => {
```
Comment (portizdev) [inline src/test/unit/views/features/public/eligibility/some-useful-info-fees.test.ts:12]: Wrong title

**Review by portizdev — CHANGES_REQUESTED** [review]

File: src/main/modules/i18n/locales/cy.json (hunk 1 above)
Comment (GorilaNaranja) [reply src/main/modules/i18n/locales/cy.json]: Good point

File: src/main/views/features/public/eligibility/some-useful-info-fees.njk (hunk 2 above)
Comment (GorilaNaranja) [reply src/main/views/features/public/eligibility/some-useful-info-fees.njk]: Nop, pasted accidentally

File: src/test/unit/views/features/public/eligibility/some-useful-info-fees.test.ts (hunk 3 above)
Comment (GorilaNaranja) [reply src/test/unit/views/features/public/eligibility/some-useful-info-fees.test.ts]: Done

**Review by portizdev — APPROVED** [review]

**Review by RosDiaf — APPROVED** [review]

**Pull request merged**